--- /opt/intel/deployment_tools/model_optimizer/mo/main.py	2021-06-22 20:30:31.000000000 +0000
+++ shared_with_container/main.py	2024-10-10 12:47:05.718612915 +0000
@@ -29,7 +29,7 @@
 from mo.utils.cli_parser import get_placeholder_shapes, get_tuple_values, get_model_name, \
     get_common_cli_options, get_caffe_cli_options, get_tf_cli_options, get_mxnet_cli_options, get_kaldi_cli_options, \
     get_onnx_cli_options, get_mean_scale_dictionary, parse_tuple_pairs, get_freeze_placeholder_values, get_meta_info, \
-    parse_transform, check_available_transforms
+    parse_transform, check_available_transforms, parse_input_value
 from mo.utils.error import Error, FrameworkError
 from mo.utils.find_ie_version import find_ie_version
 from mo.utils.get_ov_update_message import get_ov_update_message
@@ -206,7 +206,13 @@

     mean_values = parse_tuple_pairs(argv.mean_values)
     scale_values = parse_tuple_pairs(argv.scale_values)
-    mean_scale = get_mean_scale_dictionary(mean_values, scale_values, argv.input)
+
+    __input_names = []
+    for input_value in argv.input.split(','):
+        __node_name = parse_input_value(input_value)[0]
+        __input_names.append(__node_name)
+    __input_names = ",".join(__input_names)
+    mean_scale = get_mean_scale_dictionary(mean_values, scale_values, __input_names)
     argv.mean_scale_values = mean_scale

     if not os.path.exists(argv.output_dir):
